camayoc/tests/qpc/ui/utils.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+"""Utility class for UI tests."""
+
+from selenium.common.exceptions import NoSuchElementException \
```

Comment:
I would recommend avoid any rename of imported things because it is hard to ensure everyone will do the rename and also it makes hard to people not familiar with the code to read it.